backend/src/email/services/email.service.ts

```diff
@@ -296,6 +296,44 @@ const uploadProtectedCompleteOnChunk = ({
   }
 }
 
+const duplicateCampaign = async ({
+  campaignId,
+  name,
+}: {
+  campaignId: number
+  name: string
+}): Promise<Campaign | void> => {
+  const campaign = await Campaign.findByPk(campaignId)
+  if (campaign) {
+    const duplicate = await CampaignService.createCampaign({
```

Comment:
Perhaps should wrap `createCampaign` and `storeTemplate` in a transaction? Prevent the situation where a new campaign is created by the template did not get copied over.

Reply:
Haha i considered it but the transaction will now have to be unmanaged

Reply:
Let me think about it further